### PR TITLE
UPDATE/Lorem Ipsum on Catalog Pages

### LINKED ISSUE #768: [Docs] Lorem ipsum on Wasm Filter
**Current State:**
The content is filled with lorem ipsum text 
image

**Desired State:**
Content should be updated with the required information.

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by eeshaanSA**
Signed-off-by: Eeshaan Sawant <eeshaans1@gmail.com>

**Description**
Fixed an error where the new wasm filters and some of the meshery pattern cards showed placeholder texts instead of relevant information.

This PR fixes #768 

**Notes for Reviewers**
Just removed the placeholder texts, I need more iformation on the exact information that needs to be put in. Please fill me on that.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by asubedy** [comment]
@eeshaanSA you can fill `What does this pattern does?` section with information situated here. https://www.opensourceagenda.com/projects/wasm-filters

**Comment by eeshaanSA** [comment]
sure.

**Comment by eeshaanSA** [comment]
also, @asubedy, any idea what goes into the caveats and consideration section of these filters?

**Comment by eeshaanSA** [comment]
Because as of now, I can see that some filters have the info filled, but the caveats and consideration part is still filled with filler text. 
image

**Comment by eeshaanSA** [comment]
This was all the information I could find from those links, @asubedy. There are still some filters and patterns remaining.

**Comment by eeshaanSA** [comment]
Can we have this information merged, atleast?

File: collections/_catalog/wasm/metrics-store.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ compatibility:
         - 
 patternId: FILTER005
 image: /images/webassembly_logo.svg
-filterInfo: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam pulvinar placerat tellus sed posuere. Nulla venenatis fringilla faucibus. Etiam et commodo dolor, sit amet fringilla leo. Nullam tristique tristique condimentum. Maecenas sollicitudin scelerisque egestas. Suspendisse aliquet elit quis dolor gravida, et auctor ligula ornare. Nullam et sodales ante, quis varius elit. Nullam cursus, orci eleifend tristique semper, neque nisl tincidunt purus, sed ultricies felis arcu vel metus."
-filterCaveats: "llam tristique tristique condimentum. Maecenas sollicitudin scelerisque egestas. Suspendisse aliquet elit quis dolor gravida, et auctor ligula ornare. Nullam et sodales ante, quis varius elit. Nullam cursus, orci eleifend tristique semper, neque nisl tincidunt purus, sed ultricies felis arcu vel metus. "
-URL: "https://github.com/layer5io/wasm-filters/releases"
```
Comment (asubedy) [inline collections/_catalog/wasm/metrics-store.md:11]: Please add the URL field back.

**Review by asubedy — CHANGES_REQUESTED** [review]

File: collections/_catalog/wasm/metrics-store.md (hunk 1 above)
Comment (eeshaanSA) [reply collections/_catalog/wasm/metrics-store.md:11]: aight.

**Comment by eeshaanSA** [comment]
// @asubedy

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
@eeshaanSA a few merge conflicts have popped up. Will address these?

**Review by leecalcote — CHANGES_REQUESTED** [review]
merge conflicts

**Comment by eeshaanSA** [comment]
Will try to fix it asap.

**Comment by eeshaanSA** [comment]
@leecalcote @asubedy  can you check?

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**